### ISSUE TITLE
Plain text after CSV download

### ISSUE THREAD

**Issue by sachinreddykatpally** [hpcc-systems/Visualization#2734]
Hi Team,
I have a widget in DSP that has a hyperlink Address search string, but when I export it to CSV, it is coming up like this 

<a href="https://www.google.com/?gws_rd   =sslq=PO+BOX+74803+CHICAGO+
IL+60694" target= blank">PO BOX 74803 CHICAGO IL 60694</a.

The business wants to see it in plain text when they export it to CSV. Can you please look into this. 
Attaching a screenshot for the same! Let me know if you need anything else.

Regards,
Sachin. 
![address](https://user-images.githubusercontent.com/18123423/43659939-8a63bf14-972b-11e8-89f4-9c846b20d157.JPG)

**Comment by GordonSmith** [comment]
@jbrundage this is the trick I use in ECL Watch:
```
export function decodeHtml(html) {
    var txt = document.createElement("textarea");
    txt.innerHTML = html;
    return txt.value;
}
```

**Comment by jbrundage** [comment]
Merged in #2825
